src/site/controllers/WebController.php

```diff
@@ -34,14 +32,11 @@ abstract class WebController {
 
   public function __construct(
     ImmMap<string, string> $parameters,
-    private ServerRequestInterface $request,
+    private Message\IServerRequest $request,
   ) {
     $combined_params = new Map($parameters);
     foreach ($request->getQueryParams() as $key => $value) {
-      if (is_array($value)) {
-        continue;
-      }
-      $combined_params[(string)$key] = (string)$value;
+      $combined_params[$key] = $value;
     }
```

Comment:
`$request->getQueryParams()` in Nuxed returns `KeyedContainer<string, string>`, while previously it returned `KeyedContainer<arraykey, mixed>`

Nuxed normalizes all inputs using : github.com/azjezz/http-normalizer

this is done in order to avoid the use of mixed and you can be sure that you are either getting `string` or nothing, while previously it can be `string|array<arraykey, string|array<arraykey, string.......` ( `string|tree<arraykey, string>` )